### ISSUE TITLE
reconcile WCAG Tutorial Carousel with WAI ARIA Practices Carousel

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wai-tutorials#594]
I'd like to see a reconciliation between the tutorial carousel and the aria-practices carousel. 
Perhaps a discussion between the ARIA practices group and Eric.

https://www.w3.org/TR/wai-aria-practices-1.1/examples/carousel/carousel-1.html
https://www.w3.org/WAI/tutorials/carousels/

the tutorial is a few years old and perhaps update it, or propose amendments to aria practices where appropriate.

**Comment by Ryladog** [comment]
Agreed!

** katie **

*Katie Haritos-Shea*

*Principal ICT Accessibility Architect, **Board Member and W3C Advisory
Committee Rep for Knowbility *

*WCAG/Section 508/ADA/AODA/QA/FinServ/FinTech/Privacy,* *IAAP CPACC+WAS = *
*CPWA* <http://www.accessibilityassociation.org/cpwacertificants>

*Cell: **703-371-5545 <703-371-5545>** |* *ryladog@gmail.com
<ryladog@gmail.com>* *| **Oakton, VA **|* *LinkedIn Profile
<http://www.linkedin.com/in/katieharitosshea/>*

People may forget exactly what it was that you said or did, but they will
never forget how you made them feel.......

Our scars remind us of where we have been........they do not have to
dictate where we are going.

**Comment by JAWS-test** [comment]
If there are inconsistencies between APG and tutorial or errors, these should be fixed. Otherwise, I would like to keep the differences between the two, because I like the fact that there are recommendations for HTML without ARIA in the tutorial and recommendations for ARIA in the APG.

**Comment by yatil** [comment]
#### Preamble :-)

We had that thought and discussion several times on other resources and I have discussed it with @ZoeBijl in the past, too:

The audiences of both documents are different. ARIA Practices Guide basically demonstrates how to archive an outcome with ARIA, mostly disregarding the host language. Hence most structural elements are `div`s that are appropriated with ARIA roles. The tutorials follow a concept that progressively enhances HTML5 with ARIA features. One example is that without JavaScript, the Tutorial version is a list of links without any controls, while the APG version has non-functioning buttons visible. As with techniques, the solutions exist on a spectrum and the tutorial and APG versions of solutions are on different points of it. (Basically what @JAWS-test said :-D)

OTOH, the tutorials, while trying to give step-by-step examples, try to lead developers to their personal solutions instead of prescribing one. That’s why we sometimes have different examples and approaches. APG is due to its nature as a Spec-Companion much more prescriptive.

#### That said…

I think some elements are interesting for the tutorials approach as well, particularly the use of aria-`roledescription` which did not exist back in the day. If anyone has _concrete_ suggestions on what to take up, I’m happy to address them.

**Comment by ZoeBijl** [comment]
I think it would be good if Eric and I sat down and had a look at what we can copy from each other to bring them a bit closer. But as Eric already said I think it’s important for the WAI tutorials to use as much native HTML as possible.

Ironically, the one rule the ARIA Authoring Practices consistently breaks is the first rule of ARIA 🙈. With good reason of course.

**Comment by yatil** [comment]
@ZoeBijl 

> I think it would be good if Eric and I sat down and had a look at what we can copy from each other to bring them a bit closer.

Yes! Would you be open to an in-person work session at some point in… maybe November? I have a lot of notes that I want to include into the Tutorials and then work on how to addressing them with you (and eventually the rest of the ARIA gang). I think it’s probably much easier to go through the examples in person instead of async or on phone. Let me know and we can arrange something!

**Comment by ZoeBijl** [comment]
That could work! Let’s discuss the details in private.
